source/JABHandler.py

```diff
@@ -647,7 +647,11 @@ def getAccessibleTextSelectionInfo(self):
 		bridgeDll.getAccessibleTextSelectionInfo(self.vmID, self.accContext, byref(textSelectionInfo))
 		return textSelectionInfo
 
-	def _javaGetAccessibleTextRange(self, start, end):
+	def _javaGetAccessibleTextRange(self, start: int, end: int):
```

Comment:
Please add a return type.
```suggestion
	def _javaGetAccessibleTextRange(self, start: int, end: int) -> str:
```